exercism/lucians-luscious-lasagna.cs

```diff
@@ -0,0 +1,19 @@
+class Lasagna
+{
+    // TODO: define the 'ExpectedMinutesInOven()' method
+    public int ExpectedMinutesInOven(){
+        return 40;
+    }
+    // TODO: define the 'RemainingMinutesInOven()' method
+        public int RemainingMinutesInOven(int minutesInOven){
+            return ExpectedMinutesInOven() - minutesInOven;
```

Comment:
This is a really good idea, calling the `ExpectedMinutesInOven()` method here. This allows a single point of change.